components/global/history_buttons/history_buttons.tsx

```diff
@@ -3,11 +3,16 @@
 
 import React from 'react';
 import styled from 'styled-components';
+import {Tooltip} from 'react-bootstrap';
 import IconButton from '@mattermost/compass-components/components/icon-button';
 
 import {trackEvent} from 'actions/telemetry_actions';
 import * as Utils from 'utils/utils';
 import {browserHistory} from 'utils/browser_history';
+import {KEYBOARD_SHORTCUTS, KeyboardShortcutsSeq} from '../../keyboard_shortcuts/keyboard_shortcuts';
+import Constants from '../../../utils/constants';
```

Comment:
import can be shortened

```suggestion
import Constants from 'utils/constants';
```